store/postgres/src/relational/prune.rs

```diff
@@ -94,9 +97,15 @@ impl TablePair {
         if catalog::table_exists(conn, dst_nsp.as_str(), &dst.name)? {
             writeln!(query, "truncate table {};", dst.qualified_name)?;
         } else {
+            let mut list = IndexList {
+                indexes: HashMap::new(),
+            };
+            let indexes = load_indexes_from_table(conn, &src, dst_nsp.as_str())?;
```

Comment:
This should use `src_nsp` because we want to read the indexes from the source, i.e. the original of the prune.